graylog2-server/src/test/java/org/graylog2/configuration/HttpConfigurationTest.java

```diff
@@ -61,6 +62,7 @@ public void testHttpBindAddressIsIPv6Address() throws RepositoryException, Valid
     }
 
     @Test
+    @Ignore
```

Comment:
Please add a text description to the `@JsonIgnore` annotation that includes the link to the issue:

https://github.com/Graylog2/graylog2-server/issues/4459

Reply:
👍